tutorials/quick_start/pytorch_fw/ultralytics_lib/replacers.py

```diff
@@ -294,8 +103,10 @@ def val(self, data=None, **kwargs):
         overrides['rect'] = False  # rect batches as default
         overrides.update(kwargs)
         overrides['mode'] = 'val'
+        overrides['device'] = torch.device("cuda" if torch.cuda.is_available() else "cpu")
         args = get_cfg(cfg=DEFAULT_CFG, overrides=overrides)
         args.data = data or args.data
+        args.data = "coco-pose.yaml" if self.task == 'pose' else args.data
```

Comment:
I see that it is already like this throughout the quick_start code, but we might want to start fixing it now - it is better to use global constants for constant values like "coco-pose.yaml", "pose" and also "validator" below and 'device', 'mode' etc. above.